### PR TITLE
ui: fix labels when migrating instances from vmware

### LINKED ISSUE #8474: Managed VMware instances are listed as unmanaged for Import instances from VMware into a KVM cluster
##### ISSUE TYPE

 * Bug Report

##### COMPONENT NAME

~~~
VM import
~~~

##### CLOUDSTACK VERSION

~~~
cloudstack-management-4.19.0.0-SNAPSHOT.20240109
~~~

##### SUMMARY

UI lists the managed VMware instances under the unmanaged tab in UI for Import instances from VMware into a KVM cluster.

Screenshot 2024-01-09 at 2 30 23 PM

##### STEPS TO REPRODUCE

~~~
1. Add an existing VMware DC with VMs to a zone with KVM cluster.
2. Click on [Import-Export Instances], select action 'Migrate existing instances to KVM'
~~~

### PR DESCRIPTION
**Pull request by shwstppr**
### Description

Fixes #8474
Renames labels when importing from VMware

### Types of changes

- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] Enhancement (improves an existing feature and functionality)
- [ ] Cleanup (Code refactoring and cleanup, that may add test cases)
- [ ] build/CI

#### Feature/Enhancement Scale

- [ ] Major
- [ ] Minor

#### Bug Severity

- [ ] BLOCKER
- [ ] Critical
- [ ] Major
- [ ] Minor
- [ ] Trivial

### Screenshots (if appropriate):

https://github.com/apache/cloudstack/assets/153340/e94578cd-f5fa-40ea-b0e7-a5298066d890

### CODE REVIEW

**Review by nvazquez — APPROVED** [review]
Code LGTM

File: ui/src/views/tools/ManageInstances.vue [hunk 1]
```diff
@@ -322,8 +323,8 @@
             <a-col v-if="!isDiskImport" :md="24" :lg="(!isMigrateFromVmware && showManagedInstances) ? 12 : 24">
               <a-card class="instances-card">
                 <template #title>
-                  {{ $t('label.unmanaged.instances') }}
-                  <a-tooltip :title="$t('message.instances.unmanaged')">
+                  {{ (isMigrateFromVmware && vmwareVcenterType === 'existing') ? $t('label.instances') : $t('label.unmanaged.instances') }}
+                  <a-tooltip :title="(isMigrateFromVmware && vmwareVcenterType === 'existing') ? $t('message.instances.migrate.vmware') : $t('message.instances.unmanaged')">
```
Comment (DaanHoogland) [inline ui/src/views/tools/ManageInstances.vue:327]: shouldn't isMigrateFromVmware be enough condition? i.e.
```suggestion
                  {{ isMigrateFromVmware ? $t('label.instances') : $t('label.unmanaged.instances') }}
                  <a-tooltip :title="isMigrateFromVmware ? $t('message.instances.migrate.vmware') : $t('message.instances.unmanaged')">
```

**Review by rajujith — APPROVED** [review]
LGTM.
Screenshot 2024-01-11 at 10 26 49 AM

File: ui/src/views/tools/ManageInstances.vue (hunk 1 above)
Comment (shwstppr) [reply ui/src/views/tools/ManageInstances.vue:327]: @DaanHoogland current change allows showing `Instances` label only when the vCenter is an existing one. External vCenter will always considered to have unmanaged instances

Comment (DaanHoogland) [reply ui/src/views/tools/ManageInstances.vue:327]: yes but the label doesn't have to change for that, that's my point.

**Pull request merged**